core/go/internal/sequencer/originator/state_machine.go

```diff
@@ -0,0 +1,347 @@
+/*
+ * Copyright © 2025 Kaleido, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package originator
+
+import (
+	"context"
+
+	"github.com/LFDT-Paladin/paladin/common/go/pkg/i18n"
+	"github.com/LFDT-Paladin/paladin/common/go/pkg/log"
+	"github.com/LFDT-Paladin/paladin/core/internal/components"
+	"github.com/LFDT-Paladin/paladin/core/internal/msgs"
+	"github.com/LFDT-Paladin/paladin/core/internal/sequencer/common"
+	"github.com/LFDT-Paladin/paladin/core/internal/sequencer/originator/transaction"
+	"github.com/google/uuid"
+)
+
+type State int
+type EventType = common.EventType
+
+const (
+	State_Idle      State = iota //Not acting as a originator and not aware of any active coordinators
+	State_Observing              //Not acting as a originator but aware of a node (which may be the same node) acting as a coordinator
+	State_Sending                //Has some transactions that have been sent to a coordinator but not yet confirmed TODO should this be named State_Monitoring or State_Delegated or even State_Sent.  Sending sounds like it is in the process of sending the request message.
```

Comment:
`State_Inflight` maybe - agree with feedback, but not sure the proposals are much clearer.

- `State_Monitoring` - sounds identical to `State_Observing` to me
- `State_Delegated` - the observer never delegates responsibility, quite the opposite in the model (it's always responsible). It delegates transactions individually, true - but that feels very different to me.
- `State_Sent` - it might still be sending, because it's an aggregate state across everything it's doing